services/web/server/src/simcore_service_webserver/projects/projects_nodes_handlers.py

```diff
@@ -116,6 +116,21 @@ async def post_retrieve(request: web.Request) -> web.Response:
     )
 
 
+@routes.post(f"/{VTAG}/projects/{{project_uuid}}/nodes/{{node_uuid}}/restart")
```

Comment:
WARNING: these routes are still not included in the router. In the webserver we still use the old method of getting the paths from the OAS. These are once more ongoing refactoring work on the webserver ... that we need to catch up on

Reply:
thanks core cleaning it up

Reply:
ok, that is good. so they are all still in function correct? as they should. cause the node deletion currently has no such route @pcrespov

Reply:
@GitHK the entrypoint above is current not used but it will soon be re-enabled. Please can you check once more the entrypoint ... ?? 
```
@routes.post(f"/{VTAG}/projects/{{project_uuid}}/nodes/{{node_uuid}}:restart"
``` 
Is this a typo or do we still have a disagreement on these? Reviews start to be a bit redundant

Reply:
@pcrespov 🤦 never mind I got it now! There were some issues when I copied and did not check properly.